src/dgoods.cpp

```diff
@@ -566,6 +578,8 @@ extern "C" {
         auto self = receiver;
 
         if ( code == self ) {
+            bool executed = false; // TODO: REMOVE THIS
```

Comment:
```suggestion
```

Reply:
hi @kesar .. 
this line is required in order for the contract to build on the 'develop' branch of eosio.cdt  
this is due to 'EOSIO_DISPATCH_HELPER' actually being a internal macro. 
see https://github.com/EOSIO/eosio.cdt/issues/868 & https://github.com/EOSIO/eosio.cdt/pull/870